src/publish/publishToRegistry.ts

```diff
@@ -43,11 +43,15 @@ export async function publishToRegistry(originalBumpInfo: BumpInfo, options: Bea
     return publish;
   });
 
-  // if there is a prepublish hook perform a prepublish pass, calling the routine on each package
-  const prepublishHook = options.hooks?.prepublish;
-  if (prepublishHook) {
-    for (const pkg of packagesToPublish) {
-      const packageInfo = bumpInfo.packageInfos[pkg];
+
+  // finally pass through doing the actual npm publish command
+  for (const pkg of packagesToPublish) {
```

Comment:
I'm not sure if the change to order/grouping of operations here is right (but I can kind of see arguments for either ordering). @kenotron or @xugao what do you think?

Before:
prepublish all packages, publish all packages, (new) postpublish all packages

After:
for each package, prepublish/publish/postpublish

Reply:
Per the comment below I will go ahead and revert the grouping change.